src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb

```diff
@@ -502,12 +502,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 #If DEBUG Then
         ' Compile time asserts.
         Private Const s_delegateRelaxationLevelMask_AssertZero = SmallFieldMask.DelegateRelaxationLevelMask - ConversionKind.DelegateRelaxationLevelMask
-        Private _delegateRelaxationLevelMask_Assert1(s_delegateRelaxationLevelMask_AssertZero) As Boolean
-        Private _delegateRelaxationLevelMask_Assert2(-s_delegateRelaxationLevelMask_AssertZero) As Boolean
+        Private ReadOnly _delegateRelaxationLevelMask_Assert1(s_delegateRelaxationLevelMask_AssertZero) As Boolean
```

Comment:
The IDE Analyzer MakeFieldReadOnly is already enabled. Why it didn't catch these?

Reply:
The updated analyzer catches more cases.